src/platform/linux.rs

```diff
@@ -119,6 +119,41 @@ pub fn get_cursor_pos() -> Option<(i32, i32)> {
 
 pub fn reset_input_cache() {}
 
+pub fn get_focused_window_id() -> Option<i32> {
+    if let Ok(focused_window_id) = run_cmds("xdotool getactivewindow") {
```

Comment:
xdotool is not installed on my ubuntu 22.04, it works afer installation.

Reply:
Windows -> linux x11, follow menu doesn't show.
`IsMultiScreen` and `pi.isSupportMultiDisplay`  are used incorrectly.